crates/codegen/parser_templates/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "parser_templates"
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+
+[dependencies]
```

Comment:
Not sure how this PR fixes/closes #326 .. No changes to the grammar or the generated kinds.

Reply:
You changed the trivia to be not empty, this PR now assumes that those productions can fail rather than matching empty.

Reply:
Great! This enables us to remove the exclusion for `Trivia` in [this line in validation](https://github.com/NomicFoundation/slang/blob/a64a4df77661a6decb632868a92b62ec91d75cf3/crates/codegen/schema/src/validation/rules/empty_productions.rs#L35-L37). Last point in that issue.

Reply:
#337